markdown/usage/troubleshooting.md

```diff
@@ -456,17 +458,15 @@ For example, let's say it's the `markduplicates` process that is running out of
 - Within this file, add the following. Note we have increased the default `4.GB` to `16.GB`.
 
   ```nextflow
-  profiles {
-      big_data {
-        process {
-          withName: markduplicates {
-            memory = 16.GB
-          }
-        }
-      }
+  process {
+    withName: MARKDUPLICATES {
+      memory = 16.GB
+    }
   }
   ```
 
+  > ℹ️ In some cases you may need to use 'fully resolved' paths of the modules, as displayed in the nextflow console. e.g. `'NFCORE_SAREK:SAREK:MARKDUPLICATES'`, if a module is used multiple times in a workflow and not all instances need the same resources.
```

Comment:
Might be worth mentioning you should be able to find the names of most (hopefully all?) processes in the config/modules.config file within the pipeline repository? Some processes may have been imported with a different name than their `nf-core/modules` tool name.